> The pipeline “Cyrus IMAP CI Doc Gen / build (pull_request)Failing after 1m ” prints, as stated above:
> 
> ```
> Warning, treated as error:
> /__w/cyrus-imapd/cyrus-imapd/docsrc/reference/extensions/sieve.md:3:'myst' reference target not found: ../admin/sieve.rst#cyrus-imap-specific-extensions
> make: *** [Makefile:74: html] Error 2
> making "html" target exited 2 at /srv/bin/../lib/Cyrus/Docker/Command/makedocs.pm line 21.
> Error: Process completed with exit code 2.
> ```
> 
> I do not know how to get this warning disappear. I tried for some time and then gave up.

There is no tag for cyrus-imap-specific-extensions in sieve.rst.  I think you either need to create one or reference cyrus-sieve-extensions

--- a/docsrc/reference/admin/sieve.rst
+++ b/docsrc/reference/admin/sieve.rst
@@ -28,11 +28,11 @@ as a .sieve file.
 Installing Sieve
 ================
 
-This section assumes that you :ref:`compiled Cyrus <compiling>` with sieve
+This section assumes that you :ref:`compiled Cyrus <compiling>` with Sieve
 support. If you specified ``--disable-sieve`` when running ``./configure``,
 you did NOT compile the server with sieve support.
 
-Configure sieve
+Configure Sieve
 ---------------
 
 Depending on what's in your ``/etc/services`` file, sieve will usually be set
@@ -240,6 +240,8 @@ Cyrus supports a subset of these:
 
 Cyrus IMAP Specific Extensions
 ------------------------------
+
+More extensions are described at :doc:`/reference/extensions/sieve`.
 
 .. _vnd.cyrus.log:
 

--- a/docsrc/reference/extensions/sieve.md
+++ b/docsrc/reference/extensions/sieve.md
@@ -1,7 +1,8 @@
 # Sieve extensions
 
 This document lists Cyrus's nonstandard extensions to Cyrus IMAP's
-Sieve implementation, found in the `sieve/` directory.
+Sieve implementation, found in the `sieve/` directory. More extensions are
+described at [Cyrus IMAP Specific Extensions](../admin/sieve.rst#cyrus-imap-specific-extensions).
 
 ## `snooze`
 

--- a/docsrc/reference/extensions/sieve.md
+++ b/docsrc/reference/extensions/sieve.md
@@ -2,7 +2,7 @@
 
 This document lists Cyrus's nonstandard extensions to Cyrus IMAP's
 Sieve implementation, found in the `sieve/` directory. More extensions are
-described at [Cyrus IMAP Specific Extensions](../admin/sieve.rst#cyrus-imap-specific-extensions).
+described at {ref}`Cyrus IMAP Specific Extensions <cyrus-sieve-extensions>`.
 
 ## `snooze`
 

